src/lib/ppx_coda/ppx_representatives/runtime/ppx_representatives_runtime.ml

```diff
@@ -0,0 +1,27 @@
+module Util = struct
+  let rev_concat l =
+    let rec rev_concat acc l =
```

Comment:
I would name the inner loop something different. `go` is popular in the Coda sources.